### PR TITLE
Update dependancies

### PR DESCRIPTION
**Pull request by CorbanR**
Updates to scorm_engine gem

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -4,7 +4,10 @@
 require "scorm_engine"
 
 # Ensure we're picking up only the test scorm settings
-Dotenv.load(".env.test")
+Dotenv.load(".env.test.local",
+            ".env.test",
+            ".env.local",
```
Comment (mastermel) [inline spec/spec_helper.rb:9]: Curious why you added the non-test env files here? Couldn't that cause settings I have in my env.local to override what's used when running specs locally?

**Review by mastermel — CHANGES_REQUESTED** [review]
This all looks great and I appreciate you making these updates!  In particular switching over to GH Actions.

I do have one small question though on a change you made

File: .gitignore [hunk 2]
```diff
@@ -1,9 +1,10 @@
 /.bundle/
-/.env
-/.env.test
+/.env.*.local
+/.env.local
```
Comment (CalKrug) [inline .gitignore:3]: You should be able to do this with /.env*.local instead of the two lines

**Review by CalKrug — COMMENTED** [review]
1 comment.

File: spec/spec_helper.rb (hunk 1 above)
Comment (CalKrug) [reply spec/spec_helper.rb:9]: That is the same thing that is done in Learn.

Dotenv.load(".env.test.local",
            ".env.test",
            ".env.local",
            ".env")

Comment (CorbanR) [reply spec/spec_helper.rb:9]: Yeah, I purposely mimicked what learn is doing, since I expect that behavior.

File: .gitignore (hunk 2 above)
Comment (CorbanR) [reply .gitignore]: done!

File: spec/spec_helper.rb (hunk 1 above)
Comment (mastermel) [reply spec/spec_helper.rb:9]: I'm alright trying it this way. I'll just be careful when running the specs locally that I'm not bleeding in any other vars, which I can't imagine doing after the Engine migration 🤷

Comment (CorbanR) [reply spec/spec_helper.rb:9]: The documentation says `By default, load will look for a file called .env in the current working directory. Pass in multiple files and they will be loaded in order. The first value set for a variable will win.`

1st priority == .env.test.local
2nd == .env.test
3rd == .env.local

Another way of looking at is like.. `.env.local.merge(.env.test).merge(.env.test.local)`

**Review by mastermel — APPROVED** [review]

File: spec/spec_helper.rb (hunk 1 above)
Comment (mastermel) [reply spec/spec_helper.rb:9]: Oh cool! That totally alleviates my concern then 👍

**Pull request merged**